tests/drivers/adc/adc_simple/src/main.c

```diff
@@ -19,7 +19,25 @@
 /* in millisecond */
 #define SLEEPTIME  2000
 
+#if defined(CONFIG_BOARD_FRDM_K64F)
+#define ADC_DEVICE_NAME "ADC_1"
```

Comment:
Please use the config names rather than hardcoding the strings. Same as `tests/drivers/adc/adc_api/src/test_adc.c`,